compiler/noirc_frontend/src/elaborator/expressions.rs

```diff
@@ -342,14 +326,36 @@ impl<'context> Elaborator<'context> {
                     }
                 };
 
-                if func_id != FuncId::dummy_id() {
+                // Perform any check that required information from the interned function.
```

Comment:
```suggestion
```
I don't think this comment is useful, "perform any check" is too broad for me and we already have a comment on resolving generics below.